tests/test_enki.py

```diff
@@ -46,14 +52,82 @@ def get_data(cls, filename):
         path = os.path.join(cls.resource_path, filename)
         return open(path).read()
 
-
-class TestEnkiAPI(DatabaseTest, BaseEnkiTest):
-
```

Comment:
Other test classes were subclassing TestEnkiAPI, which made the tests defined in TestEnkiAPI run multiple times.